WcaOnRails/app/assets/javascripts/application.js

```diff
@@ -47,6 +48,64 @@ wca.cancelPendingAjaxAndAjax = function(id, options) {
   return wca._pendingAjaxById[id];
 };
 
+// Adopted from http://stackoverflow.com/a/21778615
+$.fn.scrollToCenter = function(speed) {
+  speed = speed || 200;
+  var el = this;
+  var elOffset = el.offset().top;
+  var windowHeight = $(window).height();
+  var offset = elOffset - Math.max((windowHeight - el.height()) / 2, 0);
+
+  $('html, body').animate({ scrollTop: offset }, speed);
+};
+
+function initMap(competitions, mapElement) {
```

Comment:
I don't think this is an acceptable name for a global method like this. Something like `initMapWithCompetitions` would be better.

How about either putting the method inside of the `wca` object (since it now is generic enough to be used by other people), or making it a jquery plugin like the `scrollToCenter` above?